### PR TITLE
First UI pass

### PR DESCRIPTION
**Pull request by bnmajor**
First pass at the Tensorboard plugin frontend

Relies on https://github.com/InsightSoftwareConsortium/itk-viewer-material-ui/pull/6

plugin

### CODE REVIEW

File: tensorboard_plugin_3d/client/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (thewtex) [inline tensorboard_plugin_3d/client/babel.config.js]: Overall, I think `.babelrc` is preferred over `babel.config.js` -- could the file be changed?

File: tensorboard_plugin_3d/client/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "version": "0.0.0",
```
Comment (thewtex) [inline tensorboard_plugin_3d/client/package.json]: Let's start with *0.0.1*.

File: tensorboard_plugin_3d/client/src/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import materialUIMachineOptions from 'itk-viewer-material-ui/src/materialUIMachineOptions'
+import modifiedCreateInterface from './main'
+
+const uiMachineOptions = { ...materialUIMachineOptions }
+
+const uiMachineActions = { ...uiMachineOptions.actions }
+uiMachineActions.createInterface = modifiedCreateInterface
+
+uiMachineOptions.actions = uiMachineActions
```
Comment (thewtex) [inline tensorboard_plugin_3d/client/src/index.js:9]: Love this!

File: tensorboard_plugin_3d/plugin.py [hunk 4]
```diff
@@ -27,14 +11,23 @@
...
 
-class ExampleRawScalarsPlugin(base_plugin.TBPlugin):
-    """Raw summary example plugin for TensorBoard."""
 
-    plugin_name = "example_raw_scalars"
+class TensorboardPlugin3D(base_plugin.TBPlugin):
+    """TensorBoard plugin for 3D rendering."""
+
+    plugin_name = "tensorboard_plugin_3d"
+    headers = [("X-Content-Type-Options", "nosniff")]
```
Comment (thewtex) [inline tensorboard_plugin_3d/plugin.py:30]: Out of curiosity, why was this needed?

**Review by thewtex — COMMENTED** [review]
🚀 🎸 

A few questions / comments inline.

File: tensorboard_plugin_3d/client/babel.config.js (hunk 1 above)
Comment (bnmajor) [reply tensorboard_plugin_3d/client/babel.config.js]: Sure thing! Commit added with this fix.

File: tensorboard_plugin_3d/client/package.json (hunk 2 above)
Comment (bnmajor) [reply tensorboard_plugin_3d/client/package.json]: Overlooked that, sorry! Fixed.

File: tensorboard_plugin_3d/plugin.py (hunk 4 above)
Comment (bnmajor) [reply tensorboard_plugin_3d/plugin.py:30]: This is to avoid a future error from Tensorboard. Without this header the following warning is currently returned:
`In 3.0, this warning will become an error: X-Content-Type-Options is required to be "nosniff"`

**Review by thewtex — APPROVED** [review]
